xarray/core/rolling.py

```diff
@@ -97,92 +95,114 @@ def __len__(self):
 
 
 class DataArrayRolling(Rolling):
-    """
-    This class adds the following class methods;
-    + _reduce_method(cls, func)
-    + _bottleneck_reduce(cls, func)
-
-    These class methods will be used to inject numpy or bottleneck function
-    by doing
-
-    >>> func = cls._reduce_method(f)
-    >>> func.__name__ = name
-    >>> setattr(cls, name, func)
-
-    in ops.inject_bottleneck_rolling_methods.
-
-    After the injection, the Rolling object will have `name` (such as `mean` or
-    `median`) methods,
-    e.g. it enables the following call,
-    >>> data.rolling().mean()
+    def __init__(self, obj, min_periods=None, center=False, **windows):
+        """
+        Moving window object for DataArray.
```

Comment:
Let's add that you should use the `DataArray.rolling()` method to construct this object instead of the class constructor.